fdmi/module.c

```diff
@@ -26,30 +26,30 @@
 #include "fdmi/module.h"
 #include "module/instance.h"
 
-static int level_fdmi_enter(struct m0_module *module);
-static void level_fdmi_leave(struct m0_module *module);
+//static int level_fdmi_enter(struct m0_module *module);
```

Comment:
No C++ style of comments.

Reply:
removed the commented line and replaced with them M0_UNUSED macro